.github/workflows/compute_md5sums.py

```diff
@@ -0,0 +1,18 @@
+import hashlib
+import json
+from pathlib import Path
+worker_dir = Path(__file__ ).parent.parent.parent / "worker"
+
+FILE_LIST = ["updater.py", "worker.py", "games.py"]
+md5dict = {}
+for file in FILE_LIST:
+    item = worker_dir / file
+    with open(item, "rb") as f:
```

Comment:
We could use the standard SRI hash already used for all our resources:
https://github.com/glinscott/fishtest/blob/c74e8ed329efc01a689a4daede7240d8456507a8/server/fishtest/__init__.py#L29-L32

Reply:
Thanks. I have no strong feeling about this. The reason I used md5sum is that the result is easy to verify using the shell command `md5sum`.

Reply:
I used read_bytes and write_text now. Thanks! If you don't mind then I will md5sum, just because the md5sum is easy to generate on the command line.